src/include/distributed/worker_protocol.h

```diff
@@ -117,7 +117,7 @@ extern bool CacheDirectoryElement(const char *filename);
 extern bool JobDirectoryElement(const char *filename);
 extern bool DirectoryExists(StringInfo directoryName);
 extern void CitusCreateDirectory(StringInfo directoryName);
-extern void CitusRemoveDirectory(StringInfo filename);
+extern void CitusRemoveDirectory(const char *filename);
```

Comment:
Also changed signature, since it seems silly to have logic which puts a `char *` into a StringInfo only to have the function only care about `filename->data`